packages/core/src/components/advanced-select/advanced-select.tsx

```diff
@@ -255,3 +256,21 @@ const RedirectElement: FunctionalComponent<{
     <dso-icon icon="chevron-right"></dso-icon>
   </a>
 );
+
+const ActiveLabelElement: FunctionalComponent<{
+  active: AdvancedSelectOption<never> | undefined;
+  options: AdvancedSelectOptionOrGroup<never>[];
+}> = ({ active, options }) => {
```

Comment:
Ik vind het fraaier als de props van een functional component hun eigen interface krijgen.

```suggestion
interface ActiveLabelElementProps {
  active: AdvancedSelectOption<never> | undefined;
  options: AdvancedSelectOptionOrGroup<never>[];
}

const ActiveLabelElement: FunctionalComponent<ActiveLabelElementProps> = ({ active, options }) => {
```

Jij?

Reply:
En eigenlijk vind ik de naam nog niet helemaal de lading dekken.

GroupActiveLabel, of ActiveGroupLabel. Ik word nooit zo warm van FunctionalComponents die eindigen op Element 🤔 Dat zijn ze toch allemaal?

Idealiter is het een UX-er die dit onderdeel benoemd: "Wat staat hier nou eigenlijk?"

Reply:
Ik heb dit gedaan: is `ActiveGroupLabel` geworden.

Heb tevens de 2 andere `Elementen` hernoemt en props interface toegevoegd.